data/incipias/incipias.txt

```diff
@@ -0,0 +1,329 @@
+
+fleet "Incipias Standard Mining"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 0
+	personality
+		timid mining harvests coward
+	variant 2
+		"Nimbus (Mining)"
+	variant 5
+		"Nimbo Stratus"
+	variant 5
+		"Nimbo Stratus" 2
+	variant 1
+		"Nimbo Stratus"
+		"Nimbus (Mining)"
+
+fleet "Incipias Standard Trade"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 50
+	personality
+		confusion 40
+		timid appeasing
+	variant 5
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbus (Civil)"
+	variant 4
+		"Nimbo Cirrus" 2
+	variant 3
+		"Gero"
+	variant 2
+		"Gero" 2
+	variant 2
+		"Gero"
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbo Cirrus" 2
+		"Nimbus (Civil)"
+
+outfitter "Hicemus"
+	"Plasma Discharger"
+	"Tractor Beam"
+	"Gas-Class Shield"
+	"Liquid-Class Shield"
+	"Liquid Hydrogen Cell"
+	"Heavy Machine Gun"
+
+	"MH Blue-Class Engine"
+	"MH Blue-Class Steering"
+	"MH Green-Class Engine"
+	"MH Green-Class Steering"
+	"MH Red-Class Engine"
+	"MH Red-Class Steering"
+
+shipyard "Hicemus"
+	"Nimbo Cirrus"
+	"Nimbo Stratus"
+	"Nimbus"
+
+news "Incipias common"
+	location
+		government "Conlatio" "Hicemus"
+	name
+		word
+			"Incipias Captain"
+			"Incipias Crew Member"
+	message
+		phrase
+			"incipias common hard defined"
+			"incipias common praise"
+
+phrase "incipias common"
+	phrase
+		"incipias common hard defined"
+		"incipias common praise"
+
+phrase "incipias common hard defined"
+	word
+		"You say you left this [system, home]? One day, [I, we] will also see other [systems, homes]."
+		"There were more [interesting, dangerous] times, to fly a spaceship, nower days a lot of people have one."
+		"I just got my license for being part of a spaceship renewed. The government controls them [strictly, like freaks]."
+		"There is nothing more [exciting, fun] to switch to turbine mode and fly through the clouds of [our home, our origin]."
+		"It has been [some, a lot of] time since the last great breakthrough has been achieved. The government [says, promises us] that they are working on something, but my guess is that they have no clue what to do."
+		"I saw a plasma discharger with a [leak, crack] not so long ago. The ship looked good from the outside, but the inside was a firestorm."
+		"I have never seen something like [you, your species]. Why don't you float like us?"
+		"I was on a mining mission not so long ago when I saw a goverment ship drop a small box. The box accelerated really fast in one direction and disappeared with a flash."
+
+phrase "incipias common praise"
+	word
+		"These technologies"
+		"These [ships, relocation mechanisms]"
+		"These [weapons, death machines]"
+		"These [engines, movement devices]"
+		"These [shields, secure mechanisms]"
+	word
+		" "
+	word
+		"of yours"
+		"you have"
+		"you have installed"
+		"of your kind"
+	word
+		" "
+	word
+		"are very impressive."
+		"are very efficient."
+		"look very stylish."
+
+phrase "incipias common disabled"
+	word
+		"Friend"
+		"[Known one, Stranger]"
```

Comment:
Aren't these oxymorons?

Reply:
I'm not sure I follow. Aren't the words in brackets supposed to be interchangeable?